spring-cloud-gcp-docs/pom.xml

```diff
@@ -90,6 +90,7 @@
 								<snippets>${basedir}/target/generated-snippets</snippets>
 								<github-tag>${github-tag}</github-tag>
 								<docinfo>true</docinfo>
+								<spring-integration-version>5.0.0.RELEASE</spring-integration-version>
```

Comment:
BTW. Doesn't it work only in IDE or at runtime when you build the project?
Maybe we can have a project property with the reference to that SI version property in Boot and use it from here as a link.
Or... just don't use SI version in Docs at all since it is managed by Boot anyway.

More over you can't use here SI-5.0 because you don't use Boot 2.0 and, therefore, SF-5.0.
And I guess you can't use until Spring Cloud 2.0 anyway. Although that raises the question if we should consider to switch to Spring Cloud 2.0/Boot 2.0 already now and develop Binder on a fresh Spring Cloud Stream foundation...

Reply:
I'll get rid of the versions after this PR.
https://github.com/spring-cloud/spring-cloud-gcp/issues/279